README.md

```diff
@@ -19,9 +20,13 @@ async fn main() -> eyre::Result<()> {
         .bytes()
         .await?;
 
-    let err = clamd_client.scan_bytes(&eicar_bytes).await.unwrap_err();
-    let msg = err.scan_error().unwrap();
-    println!("Eicar scan returned that its a virus: {}", msg);
+    let result = clamd_client.scan_bytes(&eicar_bytes).await?;
+    match result {
+        ScanResult::Malignent { infection_types } => {
+            tracing::info!("clamd found a virus(es):\n{}", infection_types.join("\n"))
```

Comment:
Should we change this to `debug` or `trace` ?

Reply:
i think debug would be better, yeah